nautobot/extras/models/models.py

```diff
@@ -448,6 +450,49 @@ def clean(self):
             self.file_extension = self.file_extension[1:]
 
 
+#
+# External integrations
+#
+
+
+class ExternalIntegration(PrimaryModel):
+    """Model for tracking integrations with external applications."""
+
+    name = models.CharField(max_length=255, unique=True)
+    remote_url = models.CharField(
+        max_length=500,
+        verbose_name="Remote URL",
+        validators=[EnhancedURLValidator()],
+    )
+    secrets_group = models.ForeignKey(
+        null=True,
+        blank=True,
+        to="extras.SecretsGroup",
+        on_delete=models.PROTECT,
+        help_text="Credentials used for authenticating with the remote system",
+    )
+    verify_ssl = models.BooleanField(
+        default=True,
+        help_text="Verify SSL certificates when connecting to the remote system",
```

Comment:
Do we need a `verbose_name="Verify SSL"` here?

Reply:
Good catch!